src/encoded/tests/test_types_protocol.py

```diff
@@ -1,10 +1,9 @@
 import datetime
 import pytest
-pytestmark = [pytest.mark.setone, pytest.mark.working, pytest.mark.schema]
 
+from ..utils import utc_today_str
```

Comment:
Why has the python formatting on github stop here? Is that just for me?

Reply:
It might be just you. GitHub was having some speed problems today. Maybe you lost a CSS file?

Reply:
No action being taken.